independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/VertxClientInputStream.java

```diff
@@ -17,6 +19,7 @@
 
 public class VertxClientInputStream extends InputStream {
 
+    private static final Logger log = LoggerFactory.getLogger(VertxClientInputStream.class);
```

Comment:
I don't think the logger is used in the end?

Reply:
Right, it got left over from something else I did